easybuild/framework/easyconfig/templates.py

```diff
@@ -299,6 +300,14 @@ def template_constant_dict(config, ignore=None, skip_lower=None, toolchain=None)
             # don't fail just because we couldn't resolve this template
             _log.warning("Failed to create mpi_cmd_prefix template, error was:\n%s", err)
 
+    # step 6. CUDA compute capabilities
+    cuda_compute_capabilities = build_option('cuda_compute_capabilities') or config.get('cuda_compute_capabilities')
+    if cuda_compute_capabilities:
+        template_values['cuda_compute_capabilities'] = ','.join(cuda_compute_capabilities)
```

Comment:
Should we rename this to `cuda_cc`?

Reply:
I'm hesitant. Using the same name as the EC param is more intuitive as we do that with other like `%(prebuildopts)s` too.

Reply:
Yeah, I agree, let's keep it as is, it's mostly cosmetic anyway and may lead to confusion.